recipes/omniscidb-cuda/recipe/build.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Make sure we are in the right place
+cd $SRC_DIR
+
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
```

Comment:
These are added by the compiler packages and is redundant.

Reply:
Disabling this LDFLAGS change results in
```
bin/initdb
error while loading shared libraries: libarrow.so.13: cannot open shared object file: No such file or directory
```

Reply:
Can you print `LDFLAGS` before this line and after?

Reply:
Before:
```
LDFLAGS= -Wl,-rpath,/usr/local/cuda-10.1.243/lib64 -Wl,-rpath-link,/usr/local/cuda-10.1.243/lib64 -L/usr/local/cuda-10.1.243/lib64
```
After:
```
LDFLAGS= -Wl,-rpath,/usr/local/cuda-10.1.243/lib64 -Wl,-rpath-link,/usr/local/cuda-10.1.243/lib64 -L/usr/local/cuda-10.1.243/lib64 -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib
```

Reply:
Hmm, that's not what I see in azure. Maybe you are using an older version of nvcc_linux-64 locally.

Reply:
```
    nvcc_linux-64:          10.1-h51cf6c1_5            conda-forge
```
This looks like the latest build.

Reply:
The script for setting the local CUDA environment uses:
```
export CUDA_HOME=$(dirname ${BASH_SOURCE[0]})
export PATH=${CUDA_HOME}/bin:${PATH}
export LD_LIBRARY_PATH=${CUDA_HOME}/lib64:${LD_LIBRARY_PATH}
export LDFLAGS="${LDFLAGS} -Wl,-rpath,${CUDA_HOME}/lib64 -Wl,-rpath-link,${CUDA_HOME}/lib64 -L${CUDA_HOME}/lib64"
```
I thought that conda-build would not pick up LDFLAGS from the environment.

Reply:
In azure, I see `export 'LDFLAGS=-Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections -Wl,-rpath,$PREFIX/lib -Wl,-rpath-link,$PREFIX/lib -L$PREFIX/lib -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib'` with ($PREFIX expanded)

Reply:
Apparently, it does, https://github.com/conda/conda-build/blob/6327c86b7b5d5e048547ccd8e1683506c78fd0f2/conda_build/environ.py#L607-L609

Reply:
Locally, I am running `conda-build` from conda `base` environment where `LDFLAGS` is unset originally. When I unset `LDFLAGS` then I see the same output as you.
So, it means that `conda-build` does not update `LDFLAGS` if it is already set.

Reply:
Can you open an issue in https://github.com/conda-forge/ctng-compiler-activation-feedstock?

Reply:
.. and the linker fails with:
```
/home/pearu/miniconda3/conda-bld/omniscidb-cuda_1572741674569/_build_env/x86_64-conda_cos6-linux-gnu/sysroot/usr/lib/librt.so: undefined reference to `__vdso_clock_gettime@GLIBC_PRIVATE'
```

Reply:
Issue opened:
https://github.com/conda-forge/ctng-compiler-activation-feedstock/issues/7

Reply:
Removed `--as-needed` "fixed" the linker failure.

Reply:
Removed redundant flags from `LDFLAGS`.